### PR TITLE
Tygron Session updates

### PR DESCRIPTION
**Pull request by PauluzzNL**
Load data when getting joinable sessions. Added joinSession functionality.

### CODE REVIEW

**Comment by HarmGriffioen** [comment]
- Set a new session type.
- param newName
-          The new session type.

I see copy pasta in your javadoc, the parameter names do not match

**Comment by nradenovic93** [comment]
Remove initialized variables

File: tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/TygronSession.java [hunk 1]
```diff
@@ -4,12 +4,29 @@
...
 
+/**
+ * TygronSession.
+ * General session handling to Tygron. A brief overview:
+ *  First you start a new session (START_NEW_SESSION)
+ *  You'll get back an ID, but you can also get the joinable session with GET_JOINABLE_SESSIONS
+ *  You can now join a session with JOIN_SESSION.
+ *  You can either close your own session with CLOSE_SESSION or 
+ *  kill the session with KILL_SESSION.
+ * @author Paul
```
Comment (devbridie) [inline tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/TygronSession.java]: Remove @author annotation

File: tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/TygronSession.java [hunk 2]
```diff
@@ -4,12 +4,29 @@
...
+ *
+ */
 public class TygronSession {
 
   private static TygronConnection apiConnection;
   private String sessionName = "";
```
Comment (devbridie) [inline tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/TygronSession.java]: Remove initialized variable.

File: tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/TygronSession.java [hunk 3]
```diff
@@ -4,12 +4,29 @@
...
+  private String sessionType;
+  private String sessionLanguage;
+  private String sessionClientToken = "";
```
Comment (devbridie) [inline tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/TygronSession.java]: Remove initialized variable.

File: tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/TygronSession.java [hunk 4]
```diff
@@ -4,12 +4,29 @@
...
+  private String sessionServerToken = "";
```
Comment (devbridie) [inline tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/TygronSession.java]: Remove initialized variable.

**Comment by nradenovic93** [comment]
Nice :+1:

**Comment by devbridie** [comment]
seems okay. :+1:

**Comment by HarmGriffioen** [comment]
Waiting on travis, if travis says :+1: then :+1:

**Pull request merged**